lib/vpn/vpn_switch.dart

```diff
@@ -59,7 +59,7 @@ class _VPNSwitchState extends State<VPNSwitch> {
     if (isMobile()) {
       return sessionModel
           .shouldShowGoogleAds((context, isGoogleAdsEnable, child) {
-        adHelper.loadAds(shouldShowGoogleAds: isGoogleAdsEnable);
+        //adHelper.loadAds(shouldShowGoogleAds: isGoogleAdsEnable);
```

Comment:
Should this line be commented out?

Reply:
Let's keep same now, i will make most of changes in username password flow.